www/app/Chat.tsx

```diff
@@ -336,9 +365,21 @@ export default function Chat({
         conversationId!,
         thoughtText
       );
-      const honchoContent = (await new Response(
-        honchoResponse
-      ).json()) as HonchoResponse;
+
+      let honchoContent: HonchoResponse;
+      if (honchoResponse instanceof Response) {
+        honchoContent = await honchoResponse.json();
+      } else {
+        // Convert ReadableStream to text first
+        const reader = honchoResponse.getReader();
+        let result = '';
+        while (true) {
+          const { done, value } = await reader.read();
+          if (done) break;
+          result += new TextDecoder().decode(value);
+        }
+        honchoContent = JSON.parse(result);
+      }
```

Comment:
We aren't streaming the Honcho response at this point so what does this all accomplish?

Reply:
Was due to verbose http code check, fixed in latest commit.